### PR TITLE
Remove password verification in admin user controller

### PR DESCRIPTION
**Pull request by luchiago**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
A little fix to remove an unused password verification, as [requested](https://github.com/thepracticaldev/dev.to/pull/5108#pullrequestreview-336178768)

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

happy new year

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -45,6 +43,8 @@ def core_params
         location
         email_public
         education
+        password
+        password_confirmation
```
Comment (rhymes) [inline app/controllers/admin/users_controller.rb]: why these?

Comment (luchiago) [reply app/controllers/admin/users_controller.rb]: I wasn't sure if the `password` params should be removed, so I left them.

Comment (rhymes) [reply app/controllers/admin/users_controller.rb]: i think you can remove them as well :) i'll ask someone else in the team to review this as well before we merge

Comment (luchiago) [reply app/controllers/admin/users_controller.rb]: Ok, I will send the change.

**Pull request merged**